idris-commands.el

```diff
@@ -74,8 +74,9 @@
 
 (defun idris-ensure-process-and-repl-buffer ()
   "Ensure that an Idris process is running and the Idris REPL buffer exists."
-  (idris-repl-buffer)
-  (idris-run))
+  (idris-run)
+  (with-current-buffer (idris-repl-buffer)
+    (goto-char (point-max))))
```

Comment:
It is intentional, but it is quite possibly not correct :) let me look at it some more then. Thanks for keeping me honest here!